source/newbiefaq.md

```diff
@@ -203,21 +192,20 @@ projects. This section includes such gems as:
 ## About the Apache Mentoring Programme
 
 <a name="NewbieFAQ-WhatistheApacheMentoringProgramme"></a>
-### What is the Apache Mentoring Programme
+### What is the Apache Mentoring Programme?
 
 Many projects are happy to assist newcomers learn about how to contribute
-to that specific project. However, some people are looking for more help, 
-or are looking for help working across Apache projects. The Mentor Programme of the The Apache Software Foundation
-provides additional support and structure for people looking to make an
+to their specific project. However, some people are looking for more help, 
+or are looking for help working across Apache projects. The ASF Mentor Programme provides additional support and structure for people looking to make an
 initial contribution to an ASF project.
 
 For more information see our [mentoring page](mentoringprogramme.html).
 
 <a name="NewbieFAQ-HowdoIApplytotheMentorProgramme?"></a>
-### How do I Apply to the Mentor Programme?
+### How do I apply to the Mentor Programme?
```

Comment:
In the headings before it's called the Mentoring Programme. Maybe we should do the same here to not confuse readers.

```suggestion
### How do I apply to the Mentoring Programme?
```